### ISSUE TITLE
default colors should be based on a palette and smart automatically applying one color of a palette at one level of a node and a complimentary one at the next level and another at the next

### ISSUE THREAD

**Issue by bluenevus** [MicroHealthLLC/mSuite#596]
so things like tree map, flow map, tree chart all have color codes and each node level represents a color set

The problem is that it doesn't change color sets and the color choices are pretty tough on the eyes.  so what we should use is a color palette and start there and be consistent in its use

top level will always default to this color the next level will always default to this, and the next and the next so that if the user never changes, the colors that are automated are 1. easy on the eyes, 2 makes sense based on node level and 3. reduces the likelihood the user even needs to change colors because its smart in color choices based on node levels

for example  https://coolors.co/a9ddd6-93c0c8-91adc2-9ba0bc-c1b8c8  because the text is black and not smart enough to change to white based on color, we need to encourage the light colors until we get the text smart enough to change black vs white based on background color

here is a minty version https://coolors.co/3c493f-7e8d85-b3bfb8-f0f7f4-a2e3c4

bold version https://coolors.co/668586-82aeb1-93c6d6-a7acd9-ab9ebd

I will say my favorite is here https://coolors.co/bcd4de-a5ccd1-e7edee-98a8ae-bec1c5  its easy on the eyes with enough contrast as a default and should be applied to all maps and whiteboard anything where we need to use one color for node 1, another color for 2nd level nodes, another color for 3rd level nodes or default colors to start such as the whiteboard can have a line #98A8AE for example.  dark enough but not black

**Comment by b-solution** [comment]
<img width="1438" alt="image" src="https://user-images.githubusercontent.com/5988861/179392664-785418ed-6d51-44db-af05-1c52cb235d97.png">
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/5988861/179392752-002eb924-f18a-44f1-97c2-84d2af8c8a9b.png">

**Comment by bluenevus** [comment]
well, I was looking for this  or for that matter any map default colors

![image](https://user-images.githubusercontent.com/3675043/179830457-99f3df11-6e14-4afd-9c0d-a253843be53d.png)

to start with this color palette.  https://coolors.co/bcd4de-a5ccd1-e7edee-98a8ae-bec1c5

**Comment by b-solution** [comment]
Hi @bluenevus, I think that you were looking some thing like this. Kindly check this and tell me if its good for you.
https://user-images.githubusercontent.com/5988861/190185765-5a4f757c-690e-4f30-b28b-09ae00e4a676.mp4

**Comment by bluenevus** [comment]
yes but to start with this color profile so that the text by default will be back and the background is light so its easy to see  https://coolors.co/bcd4de-a5ccd1-e7edee-98a8ae-bec1c5

**Comment by b-solution** [comment]
@bluenevus @LAlexMHllc I've added this color profile. Kindly test it and let me know if it is working for you.

<img width="1428" alt="image" src="https://user-images.githubusercontent.com/5988861/211775086-6d47cf16-70bf-4647-8cab-5b40f777a44f.png">

**Comment by LAlexMHllc** [comment]
Screenshot Example - TreeChart 1

![TreeChart_1](https://user-images.githubusercontent.com/11938477/213780570-1e4c2f80-ae43-447a-8a33-e7cdd03c58d5.png)

Screenshot Example - TreeChart 2

![TreeChart_2](https://user-images.githubusercontent.com/11938477/213780629-d31208ba-9f56-43fa-a2ee-40666275525e.png)

Screenshot Example - TreeChart 3

![TreeChart_3](https://user-images.githubusercontent.com/11938477/213780650-1b310b0a-822b-4d5b-aa8f-27b9852be1b1.png)

**Comment by LAlexMHllc** [comment]
Screenshot Example - TreeMap 1

![TreeMap_1](https://user-images.githubusercontent.com/11938477/213780841-2dc7e679-2c2f-43c9-85d1-6d40ca7aa826.png)

Screenshot Example - TreeMap 2

![TreeMap_2](https://user-images.githubusercontent.com/11938477/213780862-a6def92e-49be-415f-80c8-dd0a1e910e46.png)

Screenshot Example - TreeMap 3

![TreeMap_3](https://user-images.githubusercontent.com/11938477/213780893-b8d02318-1468-4f12-86ba-9ed0c774a55d.png)

**Comment by LAlexMHllc** [comment]
Reopening, since it does not satisfy the requirement.

**Comment by LAlexMHllc** [comment]
Screenshot Example - FlowMap 1

![FlowMap_1](https://user-images.githubusercontent.com/11938477/213787829-dda08e49-a3bb-4a0e-8346-431d9e00be58.png)

Screenshot Example - FlowMap 2

![FlowMap_2](https://user-images.githubusercontent.com/11938477/213787844-cbd0bdc8-3d44-4549-9730-1347b5a33988.png)

Screenshot Example - FlowMap 3

![FlowMap_3](https://user-images.githubusercontent.com/11938477/213787867-568344db-aef2-4004-8e87-7127f8e498a1.png)

**Comment by b-solution** [comment]
@LAlexMHllc Can you please give me explanation task because I did according to @bluenevus color profile default.

**Comment by LAlexMHllc** [comment]
Screenshot Example on TreeChart

![TreeChart_color_palette_example_1](https://user-images.githubusercontent.com/11938477/214341776-35a72c80-19fa-4df0-a628-560109edfa37.png)
